rpms/antigen/antigen.spec

```diff
@@ -0,0 +1,33 @@
+Name: antigen
+Version: 2.2.3
+Release: 1%{?dist}
+Summary: A plugin manager for zsh, inspired by oh-my-zsh and vundle.
+
+License: MIT
+URL: https://github.com/zsh-users/antigen
+Source0: https://github.com/zsh-users/antigen/releases/download/v%{version}/v%{version}.tar.gz
+BuildRequires: make
+Requires: zsh
+Requires: git
+
+%description
+Antigen is a small set of functions that help you easily manage your shell (zsh) plugins,
+called bundles. The concept is pretty much the same as bundles in a typical vim+pathogen setup.
+Antigen is to zsh, what Vundle is to vim.
+
+%prep
+%setup -q
+
+%install
+make build
+make PREFIX=%{buildroot}/%{_prefix} install
+
+%files
+%license LICENSE
+%{_prefix}/*
```

Comment:
%{_datadir}/antigen.zsh

If you use %{_prefix}/*,your package will own /usr/share.